.gitignore

```diff
@@ -127,3 +127,7 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# vagrant
```

Comment:
in-code comments should be used to add more relevant info.
in this case you are just repeating the line below.

either explain what's a .vagrant file is, or remove the comment.